src/index.ts

```diff
@@ -4,26 +4,34 @@ const RNUSBPrinter = NativeModules.RNUSBPrinter;
 const RNBLEPrinter = NativeModules.RNBLEPrinter;
 const RNNetPrinter = NativeModules.RNNetPrinter;
 
+export enum PrinterBrand {
+  EPSON = 'EPSON',
+  STAR = 'STAR',
+  OTHER = 'OTHER',
+}
+
 export interface PrinterOptions {
   beep?: boolean;
   cut?: boolean;
   tailingLine?: boolean;
   encoding?: string;
 }
 
-export interface IUSBPrinter {
+export interface IBasePrinter {
   device_name: string;
+  brand?: PrinterBrand;
```

Comment:
what happens if we provided brand input ?

Reply:
we switch between using epsonSDK vs GenericSDK